release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1194,6 +1198,16 @@ With this update, the `tunbr` interface is now included in the list of interface
 [id="ocp-4-11-node-bug-fixes"]
 ==== Node
 
+* Previously, the pod manager handled the registration and de-registration  of pod secrets and config maps. Because of this, pod secrets would sometimes fail to be mounted within a pod. With this fix, the pod id is included in the key that kublet uses to manage registered pods. As a result, secrets are proper mounted as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999325[*BZ#1999325*])
```

Comment:
```suggestion
* Previously, the pod manager handled the registration and de-registration of pod secrets and config maps. Because of this, pod secrets would sometimes fail to be mounted within a pod. With this fix, the pod ID is included in the key that the kubelet uses to manage registered pods. As a result, secrets are properly mounted as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999325[*BZ#1999325*])
```